src/services

```diff
@@ -168,7 +168,15 @@ p({
 })
 
 p({"class":"foam.nanos.boot.NSpec", "name":"cronScheduler",                    "lazy":false, "serviceClass":"foam.nanos.cron.CronScheduler" })
-p({"class":"foam.nanos.boot.NSpec", "name":"localUserDAO",                       "serviceScript":"return new foam.dao.PMDAO(x, new foam.dao.SequenceNumberDAO (1000, new foam.nanos.auth.PreventDuplicateEmailDAO.Builder(x).setDelegate(new foam.dao.java.JDAO(x, foam.nanos.auth.User.getOwnClassInfo(), \"users\")).build()));"})
+p({
+  "class":"foam.nanos.boot.NSpec", "name":"localUserDAO",
+  "serviceScript":"""
+    return 
+      new foam.dao.PMDAO(x, 
+      new foam.dao.SequenceNumberDAO (1000, 
+      new foam.nanos.auth.PreventDuplicateEmailDAO.Builder(x).setDelegate(
```

Comment:
Would it be reasonable to add PreventDuplicateEmailDAO to EasyDAO? @TW80000

Reply:
I would say no, since it's only applicable to models with an Email property, and we don't have an interface like `EmailAware` like we do for other properties, like `CreatedAware`. EasyDAO is generic and therefore shouldn't have options for non-generic decorators.